invenio_communities/assets/semantic-ui/js/invenio_communities/details_search/components.js

```diff
@@ -434,8 +434,12 @@ export function SearchItemCreators({ creators }) {
     );
     return link;
   }
+
   return creators.map((creator, index) => (
-    <span className="creatibutor-wrap" key={index}>
+    <span
+      className="creatibutor-wrap"
+      key={creator.person_or_org?.identifiers?.identifier}
```

Comment:
if more than one is undefined that will cause problems with the key assignment. I would try to test
```suggestion
      key={creator.person_or_org.identifiers.identifier}
```

or find another solution that ensures that the id is always unique and not a duplicate.